deploy/cf/divelog.yml

```diff
@@ -62,6 +62,21 @@ Parameters:
   AwsSecretKey:
     Description: AWS API secret key
     Type: String
+  MailFromAddress:
+    Description: The email address from which messages will be sent.
+    Type: String
+    Default: "no-reply@bottomtime.ca"
+  MailHost:
+    Description: The SMTP host through which emails will be sent.
+    Type: String
+    Default: "email-smtp.us-east-1.amazonaws.com"
+  MailUserName:
+    Description: The user name to use when authenticating with the SMTP host.
+    Type: String
+    Default: "AKIAJNVGTSUAFBYYKEUQ"
+  MailPassword:
+    Description: The password to use when authenticating with the SMTP host.
+    Type: String
```

Comment:
Also, don't forget to update the `staging.config.json` and `prod.us-east-1.config.json` files.